docs/source/HowTo/DataManagement/index.rst

```diff
@@ -0,0 +1,191 @@
+==============
+DataManagement
+==============
+
+
+For an introduction about DataManagement concepts, please see the :ref:`introduction <dms-concepts>`
+
+All the commands mentionned bellow can accept several StorageElements and LFNs as parameters. Please use `--help` for more details.
+
+.. include:: replicate.rst
+
+
+How to list one's own files
+---------------------------
+
+For a user to know its own file list::
+
+    [Dirac prod] chaen $ dirac-dms-user-lfns
+    Will search for files in /lhcb/user/c/chaen
+    /lhcb/user/c/chaen: 5 files, 2 sub-directories
+    /lhcb/user/c/chaen/GangaInputFile: 0 files, 1 sub-directories
+    /lhcb/user/c/chaen/GangaInputFile/Job_2: 1 files, 0 sub-directories
+    /lhcb/user/c/chaen/subDir: 1 files, 0 sub-directories
+    7 matched files have been put in lhcb-user-c-chaen.lfns
+
+How to see the various replicas of a file
+-----------------------------------------
+
+To list the SE where a file is stored::
+
+    [DIRAC prod] chaen $ dirac-dms-lfn-replicas /lhcb/user/c/chaen/diracTutorial.txt
+    Successful :
+        /lhcb/user/c/chaen/diracTutorial.txt :
+            CERN-USER : srm://srm-eoslhcb.cern.ch:8443/srm/v2/server?SFN=/eos/lhcb/grid/user/lhcb/user/c/chaen/diracTutorial.txt
+             RAL-USER : srm://srm-lhcb.gridpp.rl.ac.uk:8443/srm/managerv2?SFN=/castor/ads.rl.ac.uk/prod/lhcb/user/c/chaen/diracTutorial.txt
+
+
+How to get the xroot URL for my LFN
+-----------------------------------
+
+In order to get an xroot URL usable from for example ROOT::
+
+    [DIRAC prod] chaen $ dirac-dms-lfn-accessURL --Protocol=root,xroot /lhcb/user/c/chaen/diracTutorial.txt RAL-USER
```

Comment:
While trying to automate the command reference building, and looking at the `--help` output from dirac-dms-lfn-accessURL, then --Protocol is not an option, but an optional positional argument.